app_dart/lib/src/model/proto/internal/scheduler.proto

```diff
@@ -58,7 +58,7 @@ message Target {
     optional string recipe = 13;
     // Additional tags for the target.
     // The tags is list of key-value pairs that can be used to categorize different targets.
-    repeated StringPair tags = 14;
```

Comment:
Do we need to keep and mark deprecated consider how short live it is?

Reply:
My concern is there may be a dev release with tags, and we still need to support it. /cc @christopherfujino

Reply:
oh, do you mean the cocoon may point to a dev flutter branch? I thought it always grab ci.yaml from the master branch. For the cocoon itself, the only code that relies on the tags should be the request handler i added, and it is not active yet since the periodic http request is not set up yet

Reply:
Yes. Cocoon's tip of tree needs to support all ci.yamls from all supported release branches. Removing tags entirely would cause Cocoon to fail to schedule presubmit jobs, in this case `tags` would be a non-existent field. Instead, we deprecate for a few weeks then remove.

Reply:
Once a new dev release is available, we should be able to remove the entry

Reply:
unfortunately [this dev release](https://github.com/flutter/flutter/blob/dev/.ci.yaml) is going to be a beta release. I can try to cherry-pick the migration